schema/auth.graphqls

```diff
@@ -1,13 +1,16 @@
 scalar Map
 
 directive @requiresVehiclePrivilege(privileges: [Privilege!]!) on FIELD_DEFINITION | OBJECT | INTERFACE | SCALAR | ENUM
+directive @requiresManufacturerPrivilege(privilege: Privilege!) on FIELD_DEFINITION | OBJECT | INTERFACE | SCALAR | ENUM
```

Comment:
Might as well take in a list if the other one does.

Is it an OR or an AND? I don't even know.